### PR TITLE
Image height of over 80,000 px on search results

### LINKED ISSUE #423: Image height of over 80,000 px on search results of "search"
**Describe the bug**
After a couple of scrolls, the search result of query `search` gives an image which is over 80,000px in height. See the video link below

**To Reproduce**
1. Go to https://search.creativecommons.org/search?q=search&provider&li&lt&searchBy
3. Scroll down a bit.

**Screenshots**
Screenshot from 2019-07-02 22-36-23
Screenshot from 2019-07-02 22-37-50

**Desktop (please complete the following information):**
 - OS: Linux mint
 - Browser: chrome

### PR DESCRIPTION
**Pull request by Atanda1**
Fixes #423

Based on the issue, there's an image with over 8000px, and so I added `max-width: 100%;` to the class `.search-grid_image__fill`. Which allows the image fit perfectly into it's parent div regardless of the image's original height.

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -49,6 +50,8 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 'off'
```
Comment (brenoferreira) [inline .eslintrc.js]: please remove these ES lint rules changes

File: package.json [hunk 2]
```diff
@@ -113,9 +113,10 @@
     "webpack-node-externals": "^1.7.2"
   },
   "engines": {
-    "node": ">= 6.0.0",
-    "npm": ">= 3.0.0"
+    "node":">= 6.0.0",
```
Comment (brenoferreira) [inline package.json]: Please undo these changes too

File: .eslintrc.js (hunk 1 above)
Comment (Atanda1) [reply .eslintrc.js]: The disabling of the es lint rules was was an issue from https://github.com/creativecommons/cccatalog-frontend/pull/654

File: package.json (hunk 2 above)
Comment (Atanda1) [reply package.json]: Same with this one

File: .eslintrc.js (hunk 1 above)
Comment (akmadian) [reply .eslintrc.js]: Yep, but it's not within the scope of this PR. Please open a separate PR with this change.

Comment (Atanda1) [reply .eslintrc.js]: I'll reopen this PR https://github.com/creativecommons/cccatalog-frontend/pull/654 then
